domains/molai.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "This website is a link to my personal developer portfolio.",
+  "repo": "https://github.com/mtgsquad/mahir-molai",
+  "owner": {
+    "username": "mtgsquad",
+    "email": "mtgissmart@gmail.com"
+  },
+  "record": {
+    "CNAME": "mahir-molai.pages.dev"
```

Comment:
I'm hosting a personal portfolio on cloudflare pages, cloudflare pages is still in beta, so i wouldn't recommend it, but I'm trying it out, so its not a big problem.